readthedocs/search/documents.py

```diff
@@ -81,9 +81,9 @@ def faceted_search(
         }
 
         filters = {}
-        if projects_list:
+        if isinstance(projects_list, list) and projects_list:
```

Comment:
These feel like the wrong solution. I think we should just go back to the original logic. An empty list doesn't work when passed into ES either, so really we want to be checking for truthyness for it to be evaluated in a meaningful way.

Reply:
This checks that `projects_list` is effectively a list and it's not empty (`and projects_list`).